### PR TITLE
plugin,rpmqa: ignore gpg-pubkey packages

### PR DESCRIPTION
**Pull request by TomasTomecek**
beacuse these packages are automatically generated by RPM and are not
signed: koji doesn't like this

### CODE REVIEW

**Comment by jpopelka** [comment]
LGTM

File: tests/plugins/test_rpmqa.py [hunk 1]
```diff
@@ -51,10 +55,10 @@ def test_rpmqa_plugin():
     flexmock(docker.Client, logs=mock_logs)
     runner = PostBuildPluginsRunner(tasker, workflow,
                                     [{"name": PostBuildRPMqaPlugin.key,
-                                      "args": {'image_id': TEST_IMAGE}}])
+                                      "args": {'image_id': TEST_IMAGE,
+                                          "ignore_autogenerated_gpg_keys": ignore_autogenerated["ignore"]}}])
```
Comment (twaugh) [inline tests/plugins/test_rpmqa.py]: Is the indentation a bit funny here?

Comment (TomasTomecek) [inline tests/plugins/test_rpmqa.py]: whoops, looks like my vim messed up

**Comment by twaugh** [comment]
LGTM

**Pull request merged**